ZConfig/cmdline.py

```diff
@@ -155,7 +157,7 @@ def get_section_info(self, type, name):
             bk = self.basic_key(s, pos)
             if name and self._normalize_case(s) == name:
                 L.append((optpath[1:], val, pos))
-            elif bk == type:
+            elif bk == type: # pragma: no cover
```

Comment:
@mgedmin that was exactly what I thought when reading that diff (it took me a couple of minutes to stumble on the shadowing).